client/src/components/MultiJurisdictionAudit/StatusBox/index.test.tsx

```diff
@@ -699,5 +720,29 @@ describe('StatusBox', () => {
       screen.getByText('The audit has not started.')
       screen.getByText('1/2 files successfully uploaded.')
     })
+
+    it(`shows a message for full hand tally mode`, () => {
+      render(
+        <Router>
+          <JurisdictionAdminStatusBox
+            rounds={[
+              { ...roundMocks.singleIncomplete[0], isFullHandTally: true },
+            ]}
+            auditBoards={auditBoardMocks.unfinished}
+            ballotManifest={{
+              file: null,
+              processing: fileProcessingMocks.processed,
+            }}
+            batchTallies={null}
+            cvrs={null}
+            auditType="BALLOT_POLLING"
+            auditName="Test Audit"
+            isAuditOnline
+          />
+        </Router>
+      )
+      screen.getByText('Round 1 of the audit is in progress.')
+      screen.getByText('Auditing ballots.')
```

Comment:
what is this testing exactly? Are you looking for it to NOT say something?

Reply:
This was just an existing case that wasn't covered. When the audit is in full hand tally mode, this is the message displayed in the status box.